pkgs/by-name/ld/ldproxy/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, udev, stdenv }:
```

Comment:
`stdenv` is unused
```suggestion
{ lib, rustPlatform, fetchFromGitHub, pkg-config, udev }:
```

Reply:
Potentially fixed in 0cd4466c2521dbdc3f6d6da63416985877e4c403